microsoft-365/compliance/deploy-facebook-connector.md

```diff
@@ -55,13 +55,10 @@ This article contains the step-by-step process to deploy a connector that uses t
 
    ![Copy the value of the secret and save it](media/FBCimage9.png)
 
-10. Go to **Manifest** and copy the identifierUris (which is also called the AAD application Uri) as highlighted in the following screenshot. Copy the AAD application Uri to a text file or other storage location. You use it in Step 6.
-
-    ![Go to Manifest and copy the AAD application Uri](media/FBCimage10.png)
 
 ## Step 2: Deploy the connector web service from GitHub to your Azure account
 
-1. Go to [this GitHub site](https://github.com/microsoft/m365-sample-twitter-connector-csharp-aspnet) and click **Deploy to Azure**.
+1. Go to [this GitHub site](https://github.com/microsoft/m365-sample-connector-csharp-aspnet) and click **Deploy to Azure**.
 
     ![Click Deploy to Azure](media/FBCimage11.png)
```

Comment:
@markjjo did you change the screenshot as well in step 2.1 ? It was showing twitter github repo screenshot